source/documentation/virtual_machine_management_guide/topics/Opening_a_console_to_a_virtual_machine.adoc

```diff
@@ -4,6 +4,11 @@
 
 Use Remote Viewer to connect to a virtual machine.
 
+[NOTE]
+====
+To allow other users to connect to the VM, make sure you restart the virtual machine when you are finished using the console. Alternatively, the administrator can *Disable strict user checking* to eliminate the need for reboot between users. See xref:Virtual_Machine_Console_settings_explained[Virtual Machine Console Settings Explained] for more information.
```

Comment:
The same here, state 'shutdown and restart' instead of 'restart'

Reply:
Changes made.